aten/src/ATen/Declarations.cwrap

```diff
@@ -657,43 +657,6 @@
           broadcast: other inplace fallback
         - THTensor* other
 ]]
-[[
-  name: _th_remainder
-  return: argument 0
-  variants:
-    - function
-  options:
-    - cname: remainder
-      arguments:
-        - arg: THTensor* result
-          output: True
-        - THTensor* self
-        - real other
-    - cname: cremainder
-      arguments:
-        - arg: THTensor* result
-          output: True
-        - arg: THTensor* self
-          broadcast: other fallback
-        - arg: THTensor* other
-]]
-[[
-  name: _th_remainder_
-  return: argument 0
-  variants: function
-  options:
-    - cname: remainder
-      arguments:
-        - THTensor* self
-        - THTensor* self
-        - real other
-    - cname: cremainder
-      arguments:
-        - THTensor* self
-        - arg: THTensor* self
-          broadcast: other inplace fallback
```

Comment:
is it the case that currently non-broadcast-able but "same number of element" tensors work?

i.e. (2,3) X (3,2)

Reply:
No, that didn't work in TH and it doesn't work in my implementation either. They both fail with the same error:
```
>>> torch.ones([2,3]).remainder(torch.ones([3,2]))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: The size of tensor a (3) must match the size of tensor b (2) at non-singleton dimension 1
```